incubator/hnc/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Build the manager binary
-FROM golang:1.12.5 as builder
+FROM golang:1.13 as builder
```

Comment:
The current version is 1.14 - any reason not to try upgrading to the latest?

Reply:
I tried to upgrade to 1.14 but running into [this problem ](https://github.com/golang/go/issues/37436):

`runtime: mlock of signal stack failed: 1`

I took a quick look at the thread. Looks like there are some bugs in the Ubuntu kernel and they are going to include the patch in the kernel for release 20.04 ([see comment here](https://github.com/golang/go/issues/37436#issuecomment-595836976)). I am not sure how long it will take for our glinux workstation to pickup the updates.